sfyra/pkg/vm/set.go

```diff
@@ -161,7 +161,7 @@ func (set *Set) create(ctx context.Context) error {
 				},
 				PXEBooted:           true,
 				TFTPServer:          set.options.BootSource.String(),
-				IPXEBootFilename:    fmt.Sprintf("http://%s:8081/boot.ipxe", set.options.BootSource),
+				IPXEBootFilename:    "undionly.kpxe",
```

Comment:
we test new embedded script with Sfyra - despite the fact QEMU has built-in iPXE, we replace it with our own